pkg/pipeline/sink/websocket.go

```diff
@@ -53,6 +55,11 @@ func (s *WebsocketSink) Start() error {
 		for {
 			_, _, err := s.conn.ReadMessage()
 			if err != nil {
+				if _, ok := err.(*websocket.CloseError); ok {
```

Comment:
it'd be good to check against a range of diff close errors like we do [here](https://github.com/livekit/livekit/blob/master/pkg/service/rtcservice.go#L351)